scripts/vim-patch.sh

```diff
@@ -179,6 +179,16 @@ preprocess_patch() {
   # Rename path to matchit plugin.
   LC_ALL=C sed -e 's@\( [ab]/runtime\)/pack/dist/opt/matchit/\(plugin/matchit.vim\)@\1/\2@g' \
     "$file" > "$file".tmp && mv "$file".tmp "$file"
+  LC_ALL=C sed -e 's@\( [ab]/runtime\)/pack/dist/opt/matchit/\(doc/matchit.txt\)@\1/\2@g' \
+    "$file" > "$file".tmp && mv "$file".tmp "$file"
```

Comment:
We have `runtime/doc/pi_matchit.txt`. Does it need to be moved?

Reply:
No I didn't mean to imply that. I'm just commenting on your change which has `doc/matchit.txt`

Reply:
Try to keep the pattern less clever. E.g. instead of making `doc` a group and then referencing it with `\2`, just write `doc` literally in the replacement.